public/main.js

```diff
@@ -51,10 +51,32 @@ function reloadTodoList() {
         todoListPlaceholder.style.display = "none";
         todos.forEach(function(todo) {
             var listItem = document.createElement("li");
+            var deleteBtn = document.createElement("button");
             listItem.textContent = todo.title;
+            deleteBtn.textContent = "Delete";
```

Comment:
I made a couple of comments on Raul's PR here and below - could you look at that so I don't have to copy paste?